experimental/framework/fluid-static/README.md

```diff
@@ -1,5 +1,121 @@
+
+A Container instance is a organizational unit within Fluid. Each Container instance has a connection to the defined Fluid Service and contains an independent collection of collaborative objects.
+
+Containers are created and identified by unique ids. Management and storage of these ideas are the responsibility of the developer.
+
+## Using Fluid Containers
+
+Using the default `Fluid` object the developer can create and get Fluid containers. Because Fluid needs to be connected to a server containers need to be created and retrieved asynchronously.
+
+```javascript
+import Fluid from "@fluid-experimental/fluid-static";
+
+await Fluid.createContainer("_unique-id_", /* config */);
+const container = await Fluid.getContainer("_unique-id_", /* config */);
+```
+
+## Defining Fluid Containers
+
+Fluid Containers are defined by a config. The config includes initial properties of the Container as well as what collaborative objects can be dynamically created.
+
+See [`ContainerConfig`](./src/types.ts) in [`./src/types/ts`](./src/types.ts) for details about the specific properties.
+
+```javascript
+const config = {
+    name: "my-container",
+    initialObjects: {
+        /* ... */
+    },
+    dynamicDataTypes: [ /*...*/ ],
+}
+await Fluid.createContainer("_unique-id_", config);
+const container = await Fluid.getContainer("_unique-id_", config);
+```
+
+## Using initial objects
+
+The most common way to use Fluid is through initial collaborative objects that are created when the Container is created.
+
+> Note: Collaborative objects are referred to as LoadableObjects within Fluid. LoadableObjects are specific to Fluid and expose a collaborative interface. DistributedDataStructures and DataObjects are types of LoadableObjects.
+
+`initialObjects` are loaded into memory when the Container is loaded and the developer can access them off the Container via the `initialObjects` property. The `initialObjects` property has the same signature as the Container config.
+
+```javascript
+const config = {
+    name: "my-container",
+    initialObjects: {
+        map1: SharedMap,
+        pair1: KeyValueDataObject,
+    }
+}
+
+const container = await Fluid.getContainer("_unique-id_", config);
+
+const initialObjects = container.initialObjects;
+const map1 = initialObjects.map1;
+const pair1 = initialObjects["pair1"];
+```
+
+## Using dynamic objects
+
+LoadableObjects can also be created dynamically during runtime. Dynamic object types need to be defined in the  `dynamicObjectTypes` property of the ContainerConfig.
+
+The Container has a `create` method that will create a new instance of the provided type. This instance will be local to the user until attached to another LoadableObject. `initialObjects` are attached when the Container is created and should be used to store dynamic objects. When storing a LoadableObject you must store a reference to the object and not the object itself. To do this use the `handle` property on the LoadableObject.
```

Comment:
I think the "initialObjects are ... used to store dynamic objects" line is a bit confusing.  I'd suggest something like "Dynamic objects created this way should be stored in `initialObjects`, which are attached when the Container is created."